app/assets/javascripts/student_profile/student_profile_header.jsx

```diff
@@ -118,5 +119,19 @@
       );
     },
 
+    renderStudentAddress: function () {
+      const student = this.props.student;
+      const studentaddress = student.student_address;
+      if (!studentaddress) return null;
+
+      return (
+        <span>
+          {this.renderBulletSpacer()}
```

Comment:
We could actually get rid of this function and pull line 129 to 132 upward after line 57 however since we don't have dummy student address data in Fake Student class. I have decided to leave the function as is for now. I can add data to fake student class if necessary.